### ISSUE TITLE
no-mixed-spaces-and-tabs fails on blocklevel comments (regression)

### ISSUE THREAD

**Issue by aikomastboom** [eslint/eslint#4189]
this issue has returned : https://github.com/eslint/eslint/issues/1055
i noticed in eslint 1.7.1

it was fixed in this PR: https://github.com/eslint/eslint/pull/1058/files
possibly regressed by: https://github.com/eslint/eslint/commit/b8a12108f18a40bc98503f431a01879d5f6c43ca

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by alberto** [comment]
Can you post a code example exposing the error, indicating where are the tabs and spaces?

**Comment by ilyavolodin** [comment]
I can confirm that this issue is back. Here's a quick example that should fail. Tabs are indicated as `-` spaces as `_`:
```js
function test() {
-/**
-_* Some comment with a space after the tab
-_*/
-return 42;
}
```
This will return 2 errors for jsDoc comments on line 3 and 4.

**Comment by globexdesigns** [comment]
+1. I'd like to note that this is also an issue in 1.7.0 as well as 1.7.1.

**Comment by alberto** [comment]
Ok. It fails when there is a previous comment. I think I messed things up trying to ignore template literals, so I think it is related to #4190.

I'll try to fix this.

**Comment by aikomastboom** [comment]
thank you,

your test checks only the first level of indentation.. does your solution also work on comment blocks nested deeper?
